services/app/src/actions/ServerStatus.tsx

```diff
@@ -10,15 +11,27 @@ import {FetchServerStatusResponse, ServerStatusSetAction} from './ActionTypes';
 
 export function fetchServerStatus(log: any = logEvent) {
   return (dispatch: Redux.Dispatch<any>): any => {
-    return fetch(AUTH_SETTINGS.URL_BASE + '/announcements')
-      .then(handleFetchErrors)
-      .then((response: Response) => response.json())
-      .then((data: FetchServerStatusResponse) => {
-        dispatch(handleServerStatus(data));
-      }).catch((error: Error) => {
-        // Don't alert user - it's not important to them if this fails
-        log('error', 'status_fetch_err', {label: error});
-      });
+    return fetch(AUTH_SETTINGS.URL_BASE + '/announcements', {
+      retries: 2,
+      retryDelay: 1000,
+    })
+    .then(handleFetchErrors)
+    .then((response: Response) => response.json())
+    .then((data: FetchServerStatusResponse) => {
+      dispatch(handleServerStatus(data));
+    }).catch((error: Error) => {
+      const isLatestAppVersion = false;
```

Comment:
Style note: Since these values are simple and never change, we should just define them as part of the object, rather than separate variables